File: flavors/r/tests/task.R
setwd('/app')

# retrieve input parameters

library(optparse)
library(jsonlite)


option_list = list(

make_option(c("--id"), action="store", default=1, type="character", help="my description")

)

# set input parameters accordingly
opt = parse_args(OptionParser(option_list=option_list))


id <- gsub('"', '', opt$id)





names = list("Alice", "Bob")



# capturing outputs
file <- file(paste0('/tmp/names_', id, '.json'))
writeLines(toJSON(names, auto_unbox=TRUE), file)
close(file)

File: flavors/python/tests/task.py

import argparse
arg_parser = argparse.ArgumentParser()

arg_parser.add_argument('--id', action='store', type=str, required=False, dest='id', default='1')



args = arg_parser.parse_args()
print(args)

id = args.id




names = ["Alice", "Bob"]

import json
filename = "/tmp/names_" + id + ".json"
file_names = open(filename, "w")
file_names.write(json.dumps(names))
file_names.close()
